src/components/DataTable/tools/normalize.js

```diff
@@ -26,8 +26,9 @@ const normalize = (rows, headers, prevState = {}) => {
     // expansion
     rowsById[row.id] = {
       id: row.id,
-      isSelected: false,
-      isExpanded: false,
+      isSelected: row._selected || false,
```

Comment:
We typically only want to use the underscore to prefix internal methods/properties. Did you intend on leaving the underscore here?

I think keeping the same property names is cleaner and a more intuitive API.

```js
  const {id, isSelected, isExpanded, disabled} = row;
  rowsById[row.id] = {
    id,
    isSelected,
    isExpanded,
    disabled
  }
```

Reply:
Hi @vpicone ,

I guess you are right keeping underscore anyhow represents internal so having same column names are good to go. I have made changes as suggested.